File: Triangle.java
import java.util.Scanner;

public class Triangle{

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Accept three sides of the triangle
        System.out.println("Enter side 1: ");
        int side1 = scanner.nextInt();
        
        System.out.println("Enter side 2: ");
        int side2 = scanner.nextInt();
        
        System.out.println("Enter side 3: ");
        int side3 = scanner.nextInt();
        
       if(side1==10 || side2==10 || side3==10)
	{
      
            if (side1 == side2 && side2 == side3) {
                System.out.println("This is an Equilateral triangle.");
            } else if (side1 == side2 || side2 == side3 || side1 == side3) {
                System.out.println("This is an Isosceles triangle.");
            } else {
                System.out.println("This is a Scalene triangle.");
            }
        } 
		else{ System.out.println("Enter the side less then 10");
        }
    }  
     
    }

File: Triangle.py
def check_triangle_type():
    side1 = int(input("Enter side 1: "))
    side2 = int(input("Enter side 2: "))
    side3 = int(input("Enter side 3: "))
    
    if side1 <= 10 and side2 <= 10 and side3 <= 10:
        # Check for Equilateral triangle
        if side1 == side2 == side3:
            print("This is an Equilateral triangle.")

        # Check for Isosceles triangle
        elif side1 == side2 or side2 == side3 or side1 == side3:
            print("This is an Isosceles triangle.")
            
        # If neither, it must be a Scalene triangle
        else:
            print("This is a Scalene triangle.")
    else:
        print("Enter sides less than or equal to 10.")

# Main program
check_triangle_type()
